packages/vuetify/src/components/VProgressCircular/VProgressCircular.tsx

```diff
@@ -0,0 +1,142 @@
+// Styles
+import './VProgressCircular.sass'
+
+// Composables
+import { makeTagProps } from '@/composables/tag'
+import { useTheme } from '@/composables/theme'
+import { useColor } from '@/composables/color'
+
+// Utilities
+import { computed, defineComponent } from 'vue'
+import { convertToUnit, makeProps } from '@/util'
+
+export default defineComponent({
+  name: 'VProgressCircular',
+
+  props: makeProps({
+    indeterminate: Boolean,
+    rotate: {
+      type: [Number, String],
+      default: 0,
+    },
+    size: {
+      type: [Number, String],
+      default: 32,
+    },
+    color: {
+      type: String,
+      default: 'primary',
+    },
+    width: {
+      type: [Number, String],
+      default: 4,
+    },
+    value: {
+      type: [Number, String],
+      default: 0,
+    },
+    ...makeTagProps({ tag: 'div' }),
+  }),
+
+  setup (props, { slots }) {
+    const radius = 20
+    const isVisible = true
+
+    const { themeClasses } = useTheme()
+
+    const { colorClasses, colorStyles } = useColor(computed(() => ({
```

Comment:
could be `useTextColor` instead

Reply:
Done, didn't know about that part of the composable. Cool